src/main/java/org/javaparser/examples/chapter4/PrettyPrintVisitorComplete.java

```diff
@@ -19,7 +19,8 @@ public static void main(String[] args) {
         myClass.addAnnotation("MySecretAnnotation");
 
         PrettyPrinterConfiguration conf = new PrettyPrinterConfiguration();
-        conf.setIndent("  ");
+        conf.setIndentSize(2);
```

Comment:
Ouch, that's at least half a year of changes :-D

Reply:
🤷‍♂️